test.md

```diff
@@ -102,3 +102,28 @@ export PROPLET_THING_ID=""
 export PROPLET_THING_KEY=""
 propeller-proplet
 ```
+
```

Comment:
Move this to https://github.com/absmach/propeller-docs/pull/6 and delete it here